utils/eslint.sh

```diff
@@ -15,6 +15,7 @@ if [ -z "$*" ] ; then
     $(find "${WD}/tests/js/server" -name "*.js" -o -name "*.inc" | grep -v "ranges-combined") \
```

Comment:
Are the 3rdParty/rta-makedata changes desired?

Reply:
after merging https://github.com/arangodb/arangodb/pull/18628 only rta-makedata should only show the changes of this PR: https://github.com/arangodb/rta-makedata/pull/9/files
This PR is about sync'ing in a change of RTA-makedata, it doesn't have other changes than moving forward here. 
However, RTA-makedata/main is 1-dimensional, hence the step displayed here is bigger.